### PR TITLE
kmer-r1971-goolf-1.4.10.eb

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 0 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/88386de89d1ef5955533 for a full test report.

File: easybuild/easyconfigs/k/kmer/kmer-r1971-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+files_to_copy = ["Linux-amd64/*"]
+        
+sanity_check_paths = {
+    'files': ["bin/%s" % x for x in ["sim4db", "leaff", "meryl", "snapper2"]],
+    'dirs': ["lib"],
+}
+
+moduleclass = 'bio'
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/k/kmer/kmer-r1971-goolf-1.4.10.eb]: remove extra line; looks good; @boegel; test/merge?

Comment (fgeorgatos) [inline easybuild/easyconfigs/k/kmer/kmer-r1971-goolf-1.4.10.eb]: lgtm , minor changes needed here, put this in REVIEW plz!

File: easybuild/easyconfigs/k/kmer/kmer-r1971-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://sourceforge.net/code-snapshots/svn/k/km/kmer/code/']
+sources = ['%s-code-%s-trunk.zip' % (name, version[1:])]
+
+dependencies = [('bzip2', '1.0.6')]
+
+parallel = 1
+
+makeopts = ' install'
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/k/kmer/kmer-r1971-goolf-1.4.10.eb]: buildopts (and here and in many other places!)

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**